cli.js

```diff
@@ -14,14 +14,30 @@ commander
     .option('-i, --ignore <ignoredVars>', 'ignore variables, comma separated')
     .parse(process.argv);
 
-function main(args) {
+async function main(args) {
     const ignore = commander.ignore ? commander.ignore.split(',') : [];
 
     console.log('Looking for unused variables');
 
-    let unusedList = [];
+    const unusedList = [];
 
-    args.forEach(arg => {
+    const executions = args.map(path => execute(path, ignore));
+
+    const results = Promise.all(executions);
+    await results.then(results => results.forEach(result => {
+        unusedList.concat(result.value);
```

Comment:
```suggestion
        unusedList = [...unusedList, result.value];
```

This should be better?